packages/components/organisms/f-checkout/src/components/Error.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+    <card
+        is-rounded
+        has-outline
+        is-page-content-wrapper
+        card-heading-position="center"
+        data-test-id="checkout-error-page-component"
+        :class="$style['c-checkout-error']"
+    >
+        <img
+            src="../assets/images/jet-sad-bag.svg"
+            alt="">
+
+        <h1
+            :class="$style['c-checkout-error-header']"
+            data-test-id="checkout-error-page-header">
+            {{ $t('errorMessages.pageLoad.heading') }}
+        </h1>
+
+        <p
+            :class="$style['c-checkout-error-description']"
+            data-test-id="checkout-error-page-description">
+            {{ $t('errorMessages.pageLoad.description') }}
+        </p>
+    </card>
+</template>
+
+<script>
+import Card from '@justeat/f-card';
+import '@justeat/f-card/dist/f-card.css';
+
+export default {
+    components: { Card }
+};
+</script>
+
+<style lang="scss" module>
+div.c-checkout-error {
```

Comment:
Had to add the `div.` bit for more specificity.

Reply:
Could you call the class `c-checkout-errorComponent` instead? Or is that too long of a name?

Reply:
What else is overriding this? Is it possible to get around this with another class?

Reply:
@FayeCarter, the styles for the other components we have in the project don't have the `Component` suffix.

@xander-marjoram, it's the `f-card` styles overriding my styles. Did you mean to add another class that reverts/overrides those styles or add a "dummy" class just so that we have a more specific selector? It's the latter, I'd personally leave it as I have it now, but I'm more than happy to hear suggestions here! (or anywhere, hehe) 🙂

Reply:
Which properties are being overridden? If there is something specific that makes this component visually different to f-card then it probably worth its own class, which you could nest inside this one (which also helps with the extra specificity).